yt_dlp/extractor/jiocinema.py

```diff
@@ -0,0 +1,324 @@
+    _TEST = {
+        'url': 'https://www.jiocinema.com/tv-shows/cedric/2/mister-fixit/3769452',
+        'info_dict': {
+            'id': '3769452',
+            'ext': 'mp4',
+            'title': 'Mister Fixit',
+            'shortTitle': 'Mister Fixit',
+            'shortSynopsis': 'Cedric offers to fix Chen\'s broken walkman, however he realises that it is much harder than he thought it\'d be. He thinks of taking help from his father and Poppy, but th',
+            'fullSynopsis': 'Cedric offers to fix Chen\'s broken walkman, however he realises that it is much harder than he thought it\'d be. He thinks of taking help from his father and Poppy, but they get into an argument over who is a better handyman. Will Cedric be able to fix Chen\'s walkman as promised?',
+            'is4KSupported': False,
+            'is1080PSupported': False,
+            'isDolbySupported': False,
+            'introStart': 0,
+            'introEnd': 5,
+            'creditStart': 765,
+            'creditEnd': 770,
+            'recapStart': 0,
+            'recapEnd': 0,
+            'hasSubtitles': False,
+            'multiTrackAudioEnabled': False,
+            'season': '2',
+            'showName': 'Cedric',
+            'episode': '1'
+        },
+        'params': {
+            'skip_download': True,
+            'format': 'bestvideo'
+        }
+    }
+
+
+class JioCinemaMovieIE(JioCinemaBaseIE):
+    _VALID_URL = r'https?://(?:www\.)?jiocinema\.com/movies/(?P<media_name>[a-zA-Z0-9\-]*)/?(?P<season_number>[0-9]*)?/?(?P<episode_name>[a-zA-Z0-9\-]*)?/(?P<media_id>[a-zA-Z0-9\-]*)'
+    _TEST = {
+        'url': 'https://www.jiocinema.com/movies/i-choose-you-pokemon-the-movie/3777402',
+        'info_dict': {
+            'id': '3777402',
+            'ext': 'mp4',
+            'title': 'I Choose You! - Pokemon the Movie',
+            'shortTitle': 'I Choose You! - Pokemon the Movie',
+            'shortSynopsis': 'When Ash Ketchum oversleeps on his 10th birthday, he ends up with a stubborn Pikachu instead of the first partner Pokémon he wanted! But after a rocky start, Ash and Pika',
+            'fullSynopsis': 'When Ash Ketchum oversleeps on his 10th birthday, he ends up with a stubborn Pikachu instead of the first partner Pokémon he wanted! But after a rocky start, Ash and Pikachu become close friends and true partners—and when they catch a rare glimpse of the Legendary Pokémon Ho-Oh in flight, they make plans to seek it out together, guided by the Rainbow Wing it leaves behind. Trainers Verity and Sorrel join Ash on his journey. Along the way, Ash catches an abandoned Charmander, raises a Pokémon from Caterpie to Butterfree and then releases it to follow its heart, and meets the mysterious Mythical Pokémon Marshadow. When they near their goal, the arrogant Cross—Charmander\'s former Trainer—stands in their way! Can Ash and Pikachu defeat this powerful Trainer and reach Ho-Oh as they promised, or will their journey end here?',
```

Comment:
aside from changing field name to `description`, this should be MD5'd

Reply:
Fixed